hackathon.html

```diff
@@ -0,0 +1,296 @@
+---
+layout: default
+title: Hackathon
+permalink: /hackathon/
+---
+
+<div class="text-base md:text-lg">
+  <section class="hero-unit p-20 md:p-40">
+    <div class="text-center">
+      <h1 class="text-5xl md:text-8xl lg:text-9xl leading-tighter font-black">
+        Hackathon
+      </h1>
+      <p class="font-black p-3 text-2xl md:text-3xl">
+        In collaboration with
+        <a href="https://mlops.community/">MLOps Community</a>
+      </p>
+      <p class="font-black">Feb 14 - Mar 14, 2022</p>
+      <p class="p-3 text-xl font-extralight">
+        Come build MLOps and Data products, have some fun, and win exciting
+        prizes!
+      </p>
+      <button
+        class="bg-transparent hover:bg-purple-500 text-purple-700 font-semibold hover:text-white py-3 px-4 border-purple-500 border-2 hover:border-transparent rounded mt-2"
+      >
+        Join Hackathon
```

Comment:
let's add this link to the button: https://docs.google.com/forms/d/1OpnCAv2AGjDBYO5gKj3mJ0B3nrBArlfGeptNpMAlRjQ